pocket_coffea/parameters/lepton_scale_factors.yaml

```diff
@@ -95,6 +95,11 @@ lepton_scale_factors:
       "2023_preBPix": "2023PromptC"
       "2023_postBPix": "2023PromptD"
       
+    apply_ele_scale_and_smearing: 
```

Comment:
Last request: can you please add also Run2 data taking period identifiers with 'False'?